packages/authorization-process/test/getClients.test.ts

```diff
@@ -60,51 +60,77 @@ describe("getClients", async () => {
       {
         name: "test",
         userIds: [],
-        consumerId: unsafeBrandId(consumerId),
+        consumerId,
         purposeId: undefined,
       },
       { offset: 0, limit: 50 },
-      getRandomAuthData(unsafeBrandId<TenantId>(consumerId)),
+      getRandomAuthData(consumerId),
       genericLogger
     );
     expect(result.totalCount).toBe(2);
     expect(result.results).toEqual([mockClient1, mockClient2]);
   });
-  it("should get the clients if they exist (parameters: userIds)", async () => {
-    await addOneClient(mockClient3);
+  it("should get the clients if they exist (parameters: parameters userIds taken from the authData)", async () => {
+    const userId: UserId = generateId();
+
+    const mockClient7: Client = {
+      ...mockClient3,
+      users: [userId],
+    };
+    await addOneClient(mockClient7);
+
+    const result = await authorizationService.getClients(
+      {
+        name: "",
+        userIds: [generateId(), generateId()],
+        consumerId,
+        purposeId: undefined,
+      },
+      { offset: 0, limit: 50 },
+      {
+        ...getRandomAuthData(consumerId),
+        userRoles: [userRoles.SECURITY_ROLE],
+        userId,
+      },
+      genericLogger
+    );
+
+    expect(result.totalCount).toBe(1);
+    expect(result.results).toEqual([mockClient7]);
+  });
+  it("should get the clients if they exist (parameters: parameters userIds taken from the filter)", async () => {
     await addOneClient(mockClient4);
```

Comment:
I suggest to add `userIds: [userId3, userId4]` directly in this test instead of the definition of the mock, because:
- it is clear that the client contains the required values
- adding them in the definition may cause conflicts between tests
- it's difficult to keep track of which test use which value, when possible is preferable define needed data directly in the test

Reply:
Would you prefer that I do this for all tests by removing the mocks in the describe?

Reply:
no, let's just do it for this test for now